### DISCUSSION TITLE
Extra Attributes missing in number dialog

### DISCUSSION THREAD

**Discussion by BenPru** [home-assistant/frontend#14580] [Other feature requests]
Extra attributes are normally visible in the entity popup dialog:
See the binary sensor dialog "luxtronik key":
![image](https://user-images.githubusercontent.com/32298537/205942676-45c5582e-0d8b-4f6b-9847-05de79d417d4.png)

A number entity has also extra attributes:
![image](https://user-images.githubusercontent.com/32298537/205942404-016bd2f4-9b25-4f65-83f3-ed248799cc3a.png)

But they are not visible in the number entity popup dialog:
![image](https://user-images.githubusercontent.com/32298537/205943062-b2f048bf-1237-4fd2-b5c4-bc940ab48854.png)

**Comment by karwosts** [comment]
Does seem like there was a conscious decision to exclude these:

https://github.com/home-assistant/frontend/blob/6d88d46ce42d400e62da85162fbb23300d4ec38a/src/common/const.ts#L197-L207

At least that was the case for input_number, and then when number class was created it was just copied and given the same rule.
